main.go

```diff
@@ -348,6 +334,60 @@ forloop:
 	}
 }
 
+func checkCerts() (bool, error) {
+	log.Info("Checking old certs...")
+
+	secrets := []string{
+		defaults.CiliumEtcdClientTLS,
+		defaults.CiliumEtcdServerTLS,
+		defaults.CiliumEtcdPeerTLS,
+		defaults.CiliumEtcdSecrets,
+	}
+	for _, secret := range secrets {
+		_, err := k8s.Client().CoreV1().Secrets(namespace).Get(secret, meta_v1.GetOptions{})
```

Comment:
This can be done in a follow up PR, here we should verify if the certificates haven't expired as well.